### ISSUE TITLE
Debug Console: input not being read out, also arrow keys up / down are ignored

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52401]
**Environment Details:** 
VSCode Version : 1.24.0
OS Version : HighSierra10.13.5 

**Additional Details:**
ScreenReader: VoiceOver
ScreenReader Version: NVDA2018.2

**Repro Steps:**
1. Launch VS Code while VoiceOver is ON.
2.Click on View menu item--> Debug Console.

**Actual:**
 VoiceOver does not read the texts whatever we enter in the editor and also when we change its value using arrow keys. Issue also found with NVDA.

**Expected:**
VoiceOver should read whatever user enters in the editor and also should read the value which appears after pressing arrow keys.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

Attachment for Reference: 
[A11y VSCode DebugConsole VoiceOver not Reading texts in Editor..pptx](https://github.com/Microsoft/vscode/files/2118175/A11y.VSCode.DebugConsole.VoiceOver.not.Reading.texts.in.Editor.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
History navigation is not being read out by screen readers. I will add that
fyi @sandy081
